repooler.py

```diff
@@ -361,22 +383,21 @@ def generate_output(project, destid, total_lanes, req_lanes, lane_maps, acc_rati
                         else:
                             wellIndex[0] = 1
                             destNo += 1
-                            try:
-                                destid[destNo]
-                            except IndexError:
-                                print "Critical error; not enough destination plates provided"
                       
 @click.command()
 @click.option('--project_id', required=True,help='REQUIRED: ID of project to repool. Examples:P2652, P1312 etc.')
-@click.option('--dest_plate_list', default=['dp_1','dp_2','dp_3','dp_4','dp_5'], 
-              help='List of destination plates for the robot\'s csv file. Include too many rather than too few; excess will be unused Default:[dp_1,dp_2,dp_3,dp_4,dp_5]') 
+@click.option('--pool_volume', required=True,help='REQUIRED: Volume of each pool in microlitre. Typically 20 <= pool_volume <= 200')
```

Comment:
`required=True` options automatically gets a `[required]` at the end of the help text in click IIRC.